source/gloo_proxy.go

```diff
@@ -44,6 +48,11 @@ var (
 		Version:  "v1",
 		Resource: "virtualservices",
 	}
+	gatewayGVR = schema.GroupVersionResource{
```

Comment:
This may require helm rbac changes https://github.com/kubernetes-sigs/external-dns/blob/master/charts/external-dns/templates/clusterrole.yaml with helm  tests  https://github.com/kubernetes-sigs/external-dns/blob/master/charts/external-dns/tests/rbac_test.yaml